index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="styles.css">
+    </head>
+<body>
+<div class="all">
+    <header>
+        <img src="./images/My Logo.png" alt="logo holder">
+        <a href=""><img src="./images/Union.png" alt="humbergar"></a>
```

Comment:
As requested by the previous reviewer, you should use a `nav` element  to group navigational aids. The nav tag is for semantics reason, and the right position for convention and user experience for mobile devices.